### PR TITLE
Fix: IconDisplay

### PR DESCRIPTION
**Pull request by RamaChandiniDevarapalli**

### CODE REVIEW

File: apps/tenant-management-webapp/src/app/pages/admin/services/access/access.tsx [hunk 1]
```diff
@@ -30,12 +30,11 @@ export default function (): JSX.Element {
       </Main>
 
       <Aside>
-        <SupportLinks />
-
         <Gapadjustment>Helpful links</Gapadjustment>
```
Comment (athena-chen-chen) [inline apps/tenant-management-webapp/src/app/pages/admin/services/access/access.tsx:33]: Could change it to GapAdjustment and HyperlinkColor? thanks!

File: apps/tenant-management-webapp/src/app/pages/admin/services/file/file.tsx [hunk 2]
```diff
@@ -7,28 +7,29 @@ import { RootState } from '@store/index';
...
-      </a>
-      <br />
-      <a
-        rel="noopener noreferrer"
-        target="_blank"
-        href="https://github.com/GovAlta/adsp-monorepo/tree/main/apps/file-service"
-      >
-        See the code
-      </a>
+      <Gapadjustment>Helpful links</Gapadjustment>
```
Comment (athena-chen-chen) [inline apps/tenant-management-webapp/src/app/pages/admin/services/file/file.tsx:24]: If this code be used in many another place, could you make component in apps/tenant-management-webapp/src/app/components.

**Review by athena-chen-chen — CHANGES_REQUESTED** [review]

**Comment by athena-chen-chen** [comment]
Hi, Rama:

Please check 

1) The console warning still exist.

Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>.

Screen Shot 2023-08-03 at 2 59 25 PM
2) Accessibility checking. download https://www.deque.com/axe/ make sure there is no issues.

Thanks!

Athena

File: apps/tenant-management-webapp/src/app/pages/admin/services/access/access.tsx (hunk 1 above)
Comment (RamaChandiniDevarapalli) [reply apps/tenant-management-webapp/src/app/pages/admin/services/access/access.tsx]: updated

File: apps/tenant-management-webapp/src/app/pages/admin/services/file/file.tsx (hunk 2 above)
Comment (RamaChandiniDevarapalli) [reply apps/tenant-management-webapp/src/app/pages/admin/services/file/file.tsx]: refactored

**Review by athena-chen-chen — APPROVED** [review]

**Pull request merged**